core/src/main/java/com/linecorp/armeria/server/ServiceConfig.java

```diff
@@ -85,12 +86,12 @@ public final class ServiceConfig {
                   ScheduledExecutorService blockingTaskExecutor,
                   SuccessFunction successFunction,
                   Path multipartUploadsLocation, List<ShutdownSupport> shutdownSupports,
-                  HttpHeaders defaultHeaders) {
+                  HttpHeaders defaultHeaders, ServiceErrorHandler defaultServiceErrorHandler) {
```

Comment:
```suggestion
                  HttpHeaders defaultHeaders, ServiceErrorHandler serviceErrorHandler) {
```
Because it's not a default handler anymore?